tgui/packages/tgui/interfaces/BrigTimer.js

```diff
@@ -0,0 +1,121 @@
+import { Fragment } from 'inferno';
+import { useBackend } from '../backend';
+import { Button, LabeledList, Dropdown, Box, Section } from '../components';
+import { Window } from '../layouts';
+
+export const BrigTimer = (props, context) => {
+  const { act, data } = useBackend(context);
+  data.nameText = data.occupant;
+  if (data.timing) {
+    if (data.prisoner_hasrec) {
+      data.nameText = (<Box color="green">{data.occupant}</Box>);
+    } else {
+      data.nameText = (<Box color="red">{data.occupant}</Box>);
+    }
+  }
+  data.nameIcon = "pencil-alt";
+  if (data.prisoner_name) {
+    if (!data.prisoner_hasrec) {
+      data.nameIcon = "exclamation-triangle";
+    }
+  }
+  let nameOptions = [];
+  let i = 0;
+  for (i = 0; i < data.spns.length; i++) {
+    nameOptions.push(data.spns[i]);
+  }
+  return (
+    <Window resizable>
```

Comment:
Does this really need to be resizable?

Reply:
I'd rather leave it resizable. Just in case, say, someone has an astronomically long name. Or there are literally 10 people cuffed outside cells. Or similar edge situations. Its always better to allow the user to resize the window, even if you THINK they should not need to.